src/App.js

```diff
@@ -19,14 +28,20 @@ const App = () => {
     setCounterText(event.target.value)
   }
 
+  const saveCounterNameCountLocally = () =>{
+    localStorage.setItem("count",JSON.stringify(count))
+    localStorage.setItem("counterText", JSON.stringify(counterText))
+  }
```

Comment:
this should be a set of functions in a different file and imported into this one.
as the localhost/stprage functionality is not directly connected to the ui and may also be needed by other files please separate it into its own file and export it for use
also give a proper name after shifting it to another file

Reply:
put the new file under src/utils folder and name it storage.js